docs/README.md

```diff
@@ -36,7 +40,17 @@ $GLOBALS['smtgTagsProperties'] = [
 
 	// Standard meta tags
 	'keywords' => [
-		'Has keywords', 'Has another keyword'
+		'Has keywords',
+		'Has another keyword',
+		function( OutputPage $outputPage ): string {
+			// Redirects to the page are collected as keywords.
+			$redirects = $outputPage->getContext()->getTitle()->getRedirectsHere();
+			$redirect_titles = [];
+			foreach ( $redirects as $redirect ) {
+				$redirect_titles[] = $redirect->getText();
+			}
+			return implode( ', ', array_unique( $redirect_titles ));
+		}
 	],
 	'description' => 'Has some description',
```

Comment:
Most users of this extension are not programmers, thus they have no clue what a callback is. It will be great if you couls also add an example configuration for at least how adding mediawiki system messages works.

Reply:
I added another example.

Reply:
There must be a typo or something in the example. sandbox fatals after adding the example

Reply:
I have just copied both functions from the example into my home wiki settings as is, and there were no error. What does sandbox's error log say?

Reply:
Either I copied over the content of an outdated revision or did something wrong when copying over parts of the example. I had to leave so being in a hurry helped to create errors. :(

I have now the following configuratation:
```php
## Semantic Meta Tags
wfLoadExtension( 'SemanticMetaTags' );
$smtgTagsProperties = [
	'keywords' => 'Meta element keywords',
	'description' => [
		'Meta element description',
		function( OutputPage $outputPage ): string {
			global $wgLanguageCode;
			$title = $outputPage->getContext()->getTitle()->getText();
			$main_page = wfMessage( 'mainpage' )->inLanguage( $wgLanguageCode ?: 'fr' )->escaped();
			$site_name = wfMessage( 'pagetitle-view-mainpage' )->inLanguage( $wgLanguageCode ?: 'fr' )->escaped();
			$subtitle = $title !== $main_page ? wfMessage( 'tagline' )->inLanguage( $wgLanguageCode ?: 'fr' )->escaped () : '';
			return $title === $main_page ? $site_name : $title . '. ' . $subtitle;
		}
	],
];
$smtgTagsStrings = [
	'og:site_name' => 'Semantic MediaWiki - Sandbox',
	'og:image' => 'https://sandbox.semantic-mediawiki.org/w/SMWSandBoxLogo.png',
	'og:image:type' => 'image/png',
	'og:image:width' => '142',
	'og:image:height' => '142'
	];
$smtgTagsPropertyFallbackUsage = true;
```

Setting `$smtgTagsPropertyFallbackUsage = true;` had no difference for the results for test page is [20200921](https://sandbox.semantic-mediawiki.org/wiki/20200921)

Reply:
Ah, I get it now. Thanks. Works as expected.